install.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+
+set -u
+
+DOTENV_LINTER="dotenv-linter"
+DOTENV_LINTER_REPO="${DOTENV_LINTER}/${DOTENV_LINTER}"
+DOTENV_LINTER_GITHUB="https://github.com/${DOTENV_LINTER_REPO}"
+DOTENV_LINTER_RELEASE_API="https://api.github.com/repos/${DOTENV_LINTER_REPO}/releases/latest"
+
+main() {
+    require_cmd uname
+    require_cmd mkdir
+    require_cmd grep
+    require_cmd cut
+    require_cmd rm
+
+    local _target_version
+    if [ $# -eq 0 ]; then
+        get_latest_version || return 1
+        _target_version="$RETVAL"
+    else
+        _target_version="$1"
+    fi
+
+    get_architecture || return 1
+    local _arch="$RETVAL"
+
+    local _ext
+    case $_arch in
+        win-*) _ext=".zip" ;;
+        *) _ext=".tar.gz" ;;
+    esac
+
+    local _archive_name="${DOTENV_LINTER}-${_arch}${_ext}"
+    local _url="${DOTENV_LINTER_GITHUB}/releases/download/${_target_version}/${_archive_name}"
+
+    # Installation
+    local _dir="$HOME/.${DOTENV_LINTER}"
+    local _version_dir="${_dir}/${_target_version}"
+
+    if [ ! -e "$_version_dir/${DOTENV_LINTER}" ]; then
+        mkdir -p "$_version_dir"
+
+        local _archive_path="${_version_dir}/${_archive_name}"
+        download "${_url}" "${_archive_path}" || return 1
```

Comment:
Maybe we will download an archive into temp directory?

Reply:
Yes, I agree :+1: